fuzz/README.md

```diff
@@ -36,7 +33,11 @@ See [honggfuzz-rs documentation](https://github.com/rust-fuzz/honggfuzz-rs).
 cargo install afl
 ```
 
-See [the fuzz.rs book](https://fuzz.rs/book/afl/setup.html).
+Seed files should be placed in the `fuzz/fuzzer-afl/seeds/{target}/` directory, where `target` is the fuzz target name.
+
+If no seed file provided, `fuzz/fuzzer-afl/seeds/default/` will be used as seeds.
```

Comment:
Maybe we can also share seeds for different fuzzers. You can have a try (not mandatory) :)

Reply:
Examples:

https://github.com/rust-fuzz/targets/blob/master/cli.rs#L330
https://github.com/rust-fuzz/targets/blob/master/cli.rs#L226

Reply:
looks good, I will give a try